core/cmd/evm_transaction_commands_test.go

```diff
@@ -89,7 +90,8 @@ func TestShell_IndexTxAttempts(t *testing.T) {
 
 	_, from := cltest.MustAddRandomKeyToKeystore(t, app.KeyStore.Eth())
 
-	tx := cltest.MustInsertConfirmedEthTxWithLegacyAttempt(t, app.TxmStorageService(), 0, 1, from)
+	borm := cltest.NewTxStore(t, app.GetSqlxDB(), app.GetConfig().Database())
```

Comment:
How much work will it be, if we want to separate the cltest method into 2:
cltest.NewTxStore() --> for callers who can just use the regular TxStore methods
cltest.NewTestTxStore() --> for callers who want to use test methods specifically

Doing that will make it easier to isolate all places depending on the test methods.

If you feel it is too much work, then lets leave it as it is.

Reply:
Awesome, thanks!